cirq-google/cirq_google/arg_func_langs.py

```diff
@@ -121,7 +121,8 @@ def check_support(func_type: str) -> str:
         return func_type
 
     if isinstance(
-        value, (float, int, sympy.Integer, sympy.Float, sympy.Rational, sympy.NumberSymbol)
+        value,
+        (float, int, np.integer, sympy.Integer, sympy.Float, sympy.Rational, sympy.NumberSymbol),
```

Comment:
I wonder if we should make this tuple into a constant?

Reply:
I also wonder if just `number` would work? or `SupportsFloat`?

Reply:
Using `number` fails tests in `arg_func_langs.py` because `isinstance(sympy.pi, numbers.Number)` is `False`. It works in `op_serializer.py` only. `SupportsFloat` also fails tests related to that file because `issubclass(sympy.Symbol('x'), typing.SupportsFloat)` is `True` when we want it to be `False`. Should both files check types the same way or can they be different? Where should the tuple be defined as @dstrain115 suggested?

Reply:
I would just put it as a constant on the top of the arg_func_langs file.  Perhaps _ALL_NUMBER_TYPES=(float, int, np.integer,...) with a comment explaining why we need it?  I think this is optional.  I am ambivalent to whether we need to do this.